sources/pug/pages/index.pug

```diff
@@ -105,186 +159,194 @@ block page
     .container
       h6 Event Schedule
       h3 Conference Schedule
-      div Use <a target="_blank" href="https://everytimezone.com/s/8570f313">Every Time Zone</a> to translate UTC to your local time zone.
-      .tabs-custom.tabs-horizontal.tabs-corporate
-        //Tab panes
-        .tab-content.wow.fadeIn
-          .tab-pane
-            #schedule.card-group-custom.card-group-corporate(role='tablist', aria-multiselectable='false')
-              include ../components/card-content/schedule-item-intro/_schedule-item-header
-              +schedule-intro()
+      .row
+        .col-md-6.offset-md-3
+          h4(style='font-weight:normal;') CFP is still opened! We will share link soon...
+      //- div Use <a target="_blank" href="https://everytimezone.com/s/8570f313">Every Time Zone</a> to translate UTC to your local time zone.
+      //- .tabs-custom.tabs-horizontal.tabs-corporate
+      //-   //Tab panes
+      //-   .tab-content.wow.fadeIn
+      //-     .tab-pane
+      //-       #schedule.card-group-custom.card-group-corporate(role='tablist', aria-multiselectable='false')
+      //-         include ../components/card-content/schedule-item-intro/_schedule-item-header
+      //-         +schedule-intro()
 
-              +card-schedule({
-                parentId: 'schedule',
-                slug: 'paul-taylor',
-                case: 1
-              }).card-corporate
+      //-         +card-schedule({
```

Comment:
Please don't leave commented code.

Reply:
:trollface:

Reply:
oh man, in my notifications I only saw `Please don't leave commented code.` and boy, you triggered me 😆 

a true :trollface: masterpiece 🙇🏼